components/notification/NotificationCard.vue

```diff
@@ -39,6 +39,23 @@ const { notification } = defineProps<{
         <span>{{ $t("notification.signed_up") }}</span>
       </div>
     </template>
+    <template v-else-if="notification.type === 'admin.report'">
+      <NuxtLink :to="getReportRoute(notification.report?.id!)">
+        <div flex p3 items-center bg-shaded>
+          <div i-ri:flag-fill me-1 color-purple />
+          <i18n-t keypath="notification.reported">
+            <AccountDisplayName
+              :account="notification.account"
+              text-purple me-1 font-bold line-clamp-1 ws-pre-wrap break-all
+            />
+            <AccountDisplayName
+              :account="notification.report?.targetAccount!"
+              text-purple me-1 font-bold line-clamp-1 ws-pre-wrap break-all
```

Comment:
This fixes the spacing issue
```suggestion
              text-purple ms-1 font-bold line-clamp-1 ws-pre-wrap break-all
```